test/block-a-key/main.go

```diff
@@ -31,6 +32,7 @@ installation:
 usage:
   block-a-key -cert <PEM encoded x509 certificate file>
   block-a-key -jwk <JSON encoded JWK file>
+  block-a-key -privateKey <PEM encoded private key file>
```

Comment:
Nit; I know it's copied from above but believe what we want to say is:

```suggestion
  block-a-key -privateKey <PEM formatted private key file>
```